server/middleware/filter.js

```diff
@@ -5,13 +5,16 @@ const businessModel = models.Business;
 
 module.exports = {
   filter(req, res, next) {
-    const { category, location } = req.query;
+    const { category, location, name } = req.query;
     if (location) {
       return businessModel.findAll({
         where: {
           location: {
             $iLike: `%${location}%`,
-          }
+          },
+          // $or: [
```

Comment:
Hi Annmary, You started writing really clean codes until you got to this point. Please kindly clean up these commented out codes. Great work Ann!.